ratis-common/src/main/java/org/apache/ratis/datastream/SupportedDataStreamType.java

```diff
@@ -21,6 +21,7 @@
 import org.apache.ratis.util.ReflectionUtils;
 
 public enum SupportedDataStreamType implements DataStreamFactory {
+  DISABLED("org.apache.ratis.server.impl.DisabledDataStreamFactory"),
   NETTY("org.apache.ratis.netty.NettyDataStreamFactory");
```

Comment:
why not put DisabledDataStreamFactory and NettyDataStreamFactory in the same package ?

Reply:
Because of dependencies. 

`NettyDataStreamFactory` is in `ratis-netty`. A user who don't need stream server, thus choose to disable steam server, should not depend on `ratis-netty`. 

Similarly, if in the future there is a `GrpcDataStreamFactory`, it likely will be in `ratis-grpc`.